benches/themis/benches/secure_cell_wrapped_seal_master_key.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 Cossack Labs Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
+
+use themis::{keys::SymmetricKey, secure_cell::SecureCell};
+
+const CONTEXT: &[u8] = b"Themis Core benchmark";
```

Comment:
Since these are RustThemis benchmarks, might use a different context for a change ;)

Reply:
At first I thought that all benches should use the same parameters, so we can compare them. But yeah, it makes sense to add consistency :)

Reply:
Hah! I did not think about that.

Well... That makes a certain sense to keep the values the same length. I don't think a single byte will make much of a difference though.